velox/functions/sparksql/window/WindowFunctionsRegistration.cpp

```diff
@@ -15,11 +15,13 @@
  */
 #include "velox/functions/sparksql/window/WindowFunctionsRegistration.h"
 #include "velox/functions/lib/window/NthValue.h"
+#include "velox/functions/lib/window/RowNumber.h"
 
 namespace facebook::velox::functions::window::sparksql {
 
 void registerWindowFunctions(const std::string& prefix) {
   functions::window::registerIntegerNthValue(prefix + "nth_value");
+  functions::window::registerIntegerRowNumber(prefix + "row_number");
```

Comment:
Can we document this function in https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/window.rst?

Reply:
Done.